extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -400,6 +426,8 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     }
     this.ampAnalyticsConfig_ = null;
     this.qqid_ = null;
+    this.isAmpCreative_ = null;
+    return superResult;
```

Comment:
Previously we were not returning anything from unlayoutcallback, and I'm not sure why we need the return valye now. Can you explain a little why we do?

Reply:
base-element#unlayoutCallback returns a boolean (https://github.com/ampproject/amphtml/blob/master/src/base-element.js#L487)

Reply:
right, I was more curious as to why we didn't have anything being return before but now need to.

Reply:
We should have been... I'm fixing it

Reply:
gotcha, thanks. strange that the presubmits didn't catch that